scripts/us_census/acs5yr/subject_tables/s2201/process.py

```diff
@@ -0,0 +1,236 @@
+                'Native Hawaiian and Other Pacific Islander' in s or
+                'Some other race' in s):
+            continue
+
+        elif p in _FEATURE_TO_PROPERTY:
+
+            # Work status is only provided for FamilyHousehold
+            if ((p == 'No workers in past 12 months' or
+                 p == '1 worker in past 12 months' or
+                 p == '2 or more workers in past 12 months') and
+                    'Families' not in s):
+                sv.append(_FEATURE_TO_PROPERTY['Families'])
+            sv.append(_FEATURE_TO_PROPERTY[p])
+    return '_'.join(sv)
+
+
+def create_csv(output, stat_vars):
+    """Creates output CSV file."""
+    fieldnames = ['observationDate', 'observationAbout'] + stat_vars
+    with open(output, 'w') as f_out:
+        writer = csv.DictWriter(f_out, fieldnames=fieldnames)
+        writer.writeheader()
+
+
+def write_csv(filename, reader, output, stat_vars):
+    """Reads input_file and writes cleaned CSV to output."""
+    if 'ACSST5Y' not in filename:
+        return
+    fieldnames = ['observationDate', 'observationAbout'] + stat_vars
+    stat_var_set = set(stat_vars)
+    with open(output, 'a') as f_out:
+        writer = csv.DictWriter(f_out, fieldnames=fieldnames)
+        observation_date = filename.split('ACSST5Y')[1][:4]
+        valid_columns = {}
+        for row in reader:
+            if row['GEO_ID'] == 'id':
+
+                # Map feature names to stat vars
+                for c in row:
+                    sv = convert_column_to_stat_var(row[c])
+                    if sv in stat_var_set:
+                        valid_columns[c] = sv
+                continue
+
+            new_row = {
+                'observationDate': observation_date,
+                'observationAbout': 'dcid:geoId/' + row['GEO_ID'].split('US')[1]
+            }
+            for c in row:
+
+                # We currently only support the stat vars in the list
+                if c not in valid_columns:
+                    continue
+                sv = valid_columns[c]
+
+                # Exclude missing values
+                if (row[c] == '**' or row[c] == '-' or row[c] == '***' or
+                        row[c] == '*****' or row[c] == 'N' or row[c] == '(X)' or
+                        row[c] == 'null'):
+                    continue
+
+                # Exclude percentages
+                if '.' in row[c]:
+                    continue
+
+                # Exclude suffix from median values
+                if (row[c][-1] == '-' or row[c][-1] == '+'):
+                    new_row[sv] = row[c][:-1]
+                else:
+                    new_row[sv] = row[c]
+            writer.writerow(new_row)
+
+
+def create_tmcf(output, stat_vars):
+    """Writes tMCF to output."""
+    with open(output, 'w') as f_out:
+        for i in range(len(stat_vars)):
+            unit = ''
+            if 'Income' in stat_vars[i]:
+                unit = _UNIT_TEMPLATE
```

Comment:
Maybe this can be part of the input map too: column-name => (sv-id, unit)

Reply:
added to input map